### DISCUSSION TITLE
getCurrentInstance API的使用

### DISCUSSION THREAD

**Discussion by Taxpolat** [NervJS/taro#11232] [Q & A]
在Vue3下，如果出现需要使用vue3的getCurrentInstance API 即需要使用Taro框架的getCurrentInstance API的情况怎么处理？

**Comment by Chen-jj** [comment]
也是一样使用呀：
![image](https://user-images.githubusercontent.com/11807297/154826925-0de7b139-3021-453b-9412-834ba56f66fc.png)
